src/ngx_http_lua_headers_in.c

```diff
@@ -645,14 +645,95 @@ ngx_http_clear_builtin_header(ngx_http_request_t *r,
 }
 
 
+static ngx_hash_init_t  lua_req_header_hash;
+static u_char           header_string[512];
```

Comment:
1. the ngx_cycle.conf_ctx is 0 when I call ngx_http_cycle_get_module_main_conf in this init func, unless choose another place to init
2. ngx_http_get_module_main_conf, no request argument
3. ngx_http_conf_get_module_main_conf, no cf argument